docs/json.md

```diff
@@ -65,6 +65,8 @@ Each JSON Schema provided in this repository is automatically generated. For thi
 
 The ABAP types are self-contained, so it is possible to work on them in any system (e.g., in an SAP BTP, ABAP environment system).
 
+Instead of hashed tables, please use sorted tables with a defined key.
```

Comment:
I am wondering whether we should add the information to the sub chapter "Type Mapping" below?

There we could also enhance the table. What do you think?

Reply:
I enhanced the table. See my newest commit. Please rereview